src/reduceRight.js

```diff
@@ -20,24 +20,42 @@ var _curry3 = require('./internal/_curry3');
  * @memberOf R
  * @since v0.1.0
  * @category List
- * @sig (a,b -> a) -> a -> [b] -> a
- * @param {Function} fn The iterator function. Receives two values, the accumulator and the
- *        current element from the array.
+ * @sig (a, b -> b) -> b -> [a] -> b
+ * @param {Function} fn The iterator function. Receives two values, the current element from the array
+ *        and the accumulator.
  * @param {*} acc The accumulator value.
  * @param {Array} list The list to iterate over.
  * @return {*} The final, accumulated value.
- * @see R.addIndex
+ * @see R.reduce, R.addIndex
  * @example
  *
- *      var pairs = [ ['a', 1], ['b', 2], ['c', 3] ];
- *      var flattenPairs = (acc, pair) => acc.concat(pair);
+ *      R.reduceRight(R.subtract, 0, [1, 2, 3, 4]) // => (1 - (2 - (3 - (4 - 0)))) = -2
+ *              2
+ *               \
+ *            1 - 3
+ *                 \
+ *              2 - -1
+ *                   \
+ *                3 - 4
+ *                     \
+ *                  4 - 0
+ *
+ *      R.reduce(R.subtract, 0, [1, 2, 3, 4]) // => ((((0 - 1) - 2) - 3) - 4) = -10
+ *                  -10
+ *                 /
+ *                -6 - 4
+ *               /
+ *              -3 - 3
+ *             /
+ *            -1 - 2
+ *           /
+ *          0 - 1
```

Comment:
ok, I'll modify it, and add this comment to `reduce`.

I also changed the `mapAccumRight` iterator function arguments' order, can you and @paldepind review that?